sympy/geometry/polygon.py

```diff
@@ -374,6 +374,63 @@ def centroid(self):
             cy += v*(y1 + y2)
         return Point(simplify(A*cx), simplify(A*cy))
 
+
+    def second_moment_of_area(self, p=None):
+        """Returns the second moments and products of area of a two dimensional polygon.
```

Comment:
Why explicit mention of "two dimensional", a polygon is always 2-D?

Reply:
it was suggested by @moorepants and the definition of Polygon I know is a plane bounded by at least three straight sides and angles. so it is possible that we can have a polygon in 3d.